test/verify/check-storage-btrfs

```diff
@@ -445,6 +445,7 @@ class TestStorageBtrfs(storagelib.StorageCase):
         self.click_dropdown(self.card_row("Storage", name=label) + " + tr", "Mount")
         self.dialog({"mount_point": mount_point})
         b.wait_visible(self.card_row("Storage", location=mount_point))
+        self.addCleanup(m.execute, f"while mountpoint -q {mount_point} && ! umount {mount_point}; do sleep 0.2; done;")
```

Comment:
Ah, so the `fuser` loop in testlib's `nonDestructiveSetup()` doesn't catch this? :cry:

Reply:
Yeah hmm can't find the old status but fuser --mount didn't catch it for some reason